woocommerce/woocommerce-android
# Free Trial: Support report subscription issue action

Summary
==========
Fix issue #8454 by linking the `Report subscription issue` button with the Support Request form. The `Upgrades` Help origin was added alongside a `Free trial` tag to identify the issue scenario inside the Zendesk ticket.

As discussed internally (Slack C02KUCFCSFP/p1678847948565039), we will use the already available help options, so there's no action needed on the Support request form side.

Capture
==========
https://user-images.githubusercontent.com/5920403/225208367-f433d813-283e-4455-b23b-5fbfd370cdae.mp4

<img width="300" alt="image" src="https://user-images.githubusercontent.com/5920403/225208869-02d3700f-7412-4fb1-86c8-21f0b8ef3dd8.png">

How to Test
==========
1. Open the app and go to the menu view
2. Hit the `Upgrades` section
3. Click on the `Report subscription issue` button
4. Verify that the Support Request Form is open as expected
5. Create a ticket using the `Mobile app` section and verify if the ticket is created showing the `origin:upgrades` and `trial` tag only appear when the site is in free trial.

Update release notes:
